org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/part/WindowContainer.java

```diff
@@ -709,35 +709,28 @@ public Tab insertAfter(Tab refTab, Component comp, String title, boolean closeab
      * @return org.zkoss.zul.Tab
      */
     public org.zkoss.zul.Tab replace(org.zkoss.zul.Tab refTab, Window comp, String title) {
-    	 
-         if (refTab == null)  
-         {
-         	throw new IllegalArgumentException();
-         }
-         else
-         {
-         	org.zkoss.zul.Tabpanel refpanel = refTab.getLinkedPanel();
-         	Component firstChild = refpanel.getFirstChild();
-         	if(firstChild instanceof Window) {
-	     		if(firstChild instanceof ProcessDialog)
-	     			((ProcessDialog)firstChild).unlockUI(null);
-	     		else if(firstChild instanceof ZkReportViewer)
-					((ZkReportViewer)firstChild).hideBusyMask();
-				else if(firstChild instanceof AbstractADWindowContent)
-					((AbstractADWindowContent)firstChild).hideBusyMask();
-	     		((Window) firstChild).onClose();
-	     		comp.setParent(refpanel);
-         	}
-         	else {
-         		firstChild.detach();
-         		comp.setParent(refpanel);
-         	}
-         }
-         if (title != null) 
-         {
- 	        setTabTitle(title, refTab);
-         }
-        return refTab;
+		org.zkoss.zul.Tabpanel refpanel = refTab.getLinkedPanel();
+		Component firstChild = refpanel.getFirstChild();
+		if(firstChild instanceof Window) {
+			if(firstChild instanceof ProcessDialog)
+				((ProcessDialog)firstChild).unlockUI(null);
+			else if(firstChild instanceof ZkReportViewer)
+				((ZkReportViewer)firstChild).hideBusyMask();
+			else if(firstChild instanceof AbstractADWindowContent)
+				((AbstractADWindowContent)firstChild).hideBusyMask();
+			((Window) firstChild).onClose();
+			comp.setParent(refpanel);
+		}
+		else {
+			firstChild.detach();
+			comp.setParent(refpanel);
+		}
+		
+		if (title != null) 
+		{
+			setTabTitle(title, refTab);
+		}
+		return refTab;
```

Comment:
This change seems unnecessary.

Reply:
@hengsin about `replace` method?

Reply:
yes, the replace method

Reply:
@hengsin done!

Reply:
hi @MarcosGOFavaretto , if you are not making any changes to WindowContainer.java, you should just revert all changes to it. As it is, the change set includes many white space changes to WindowContainer.java which is undesirable.